src/Dropdown/Dropdown.js

```diff
@@ -71,20 +78,22 @@ export default function Dropdown({
   useEffect(() => {
     if (isOpen) {
       setTimeout(() => {
-        menuRef.current.focus();
+        if (menuRef.current) menuRef.current.focus();
       });
     }
   }, [isOpen]);
 
   useKeyPress('Escape', () => {
     if (isOpen) {
+      triggerRef.current.focus();
```

Comment:
We manually implement `returnFocus` here, so that inputs can be directly focused (without a second click), which was prevented by using FocusLock's `returnFocus`.